phpBB/includes/acp/acp_extensions.php

```diff
@@ -65,22 +77,22 @@ function main()
 		* @var	string	ext_name		Extension name from request
 		* @var	int		safe_time_limit	Safe limit of execution time
 		* @var	int		start_time		Start time
+		* @var	string	tpl_name		Template file to load; leave empty to continue execution, filled in if ready to finish
 		* @since 3.1.11-RC1
 		*/
 		$u_action = $this->u_action;
-		$vars = array('action', 'u_action', 'ext_name', 'safe_time_limit', 'start_time');
+		$tpl_name = '';
+		$vars = array('action', 'u_action', 'ext_name', 'safe_time_limit', 'start_time', 'tpl_name');
 		extract($this->phpbb_dispatcher->trigger_event('core.acp_extensions_run_action', compact($vars)));
```

Comment:
Ok, will change it, clearer, but, how do I document this change?
@since 3.2.0 original name (already published in its current form)
@changed 3.2.1 rename (would change for 3.2.1 with new name)
@since 3.1.11 new name (new name is first name for 3.1 branch)
Note that these also applies even if the name is not changed: the changes of the checks (before and after the event), additional param tpl_name, action none should be documented for 3.2.
How do I add them?